src/test/java/org/minions/devfund/benjamin/MovieDatabaseTest.java

```diff
@@ -0,0 +1,118 @@
+package org.minions.devfund.benjamin;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * Class to test MovieDatabase class.
+ */
+public class MovieDatabaseTest {
+    /**
+     * Test Add a new movie and actors.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Batman Forever" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)